File: 01. Document Viewer/04. ASP.NET/ASP.NET MVC/VB/Controllers/DocumentViewerController.vb
Imports System.Web.Mvc
Imports Gnostice.Controls.ASP

Public Class DocumentViewerController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult

        Dim vc As ViewerController = New ViewerController()

        Dim docPath As String = Server.MapPath("~\\App_Data\\") + "html5documentviewerarticle.pdf"

        Dim docURI As String = vc.LoadDocument(docPath)

        ViewBag.documentURI = docURI

        Return View()
    End Function

End Class

File: 01. Document Viewer/04. ASP.NET/ASP.NET MVC/C#/Controllers/DocumentViewerController.cs
using Gnostice.Controls.ASP;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace ASP.NET_MVC_Razor_CS_Document_Viewer.Controllers
{
    public class DocumentViewerController : Controller
    {
        //
        // GET: /DocumentViewer/
        public ActionResult Index()
        {
            string documentpath = Server.MapPath(".") + "\\App_Data\\sampleFormDocument.pdf";
            ViewerController vc = new ViewerController();
            string documentURI=vc.LoadDocument(documentpath);
            ViewBag.docURI = documentURI;
            return View();
        }

    }
}
